packages/conf-libclang/conf-libclang.11/files/configure.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash -ex
+
+find_llvm_config () {
+    # Locate llvm-config (taken from conf-llvm's configure.sh file)
+
+    shopt -s nullglob
+    for version in 11 10 9 8 7 6 5 4 3; do
+        if hash brew 2>/dev/null; then
+           brew_llvm_config="$(brew --cellar)"/llvm*/${version}*/bin/llvm-config
+        fi
+        for llvm_config in \
+            llvm-config-${version} llvm-config-${version}.0 \
+            llvm-config${version}0 llvm-config${version} \
+            llvm-config-mp-$version \
+            llvm-config-mp-${version}.0 $brew_llvm_config \
+            /usr/lib64/llvm/${version}/bin/llvm-config \
+            /usr/lib/llvm/${version}/bin/llvm-config \
+            llvm-config; do
+            llvm_version="`$llvm_config --version`" || continue
```

Comment:
it is to be noted that on fedora, centos and similar distributions, `llvm-config` is named `llvm-config-64` (on 64bits arch) and `llvm-config-32` (on 32bits arch), see https://github.com/ocaml/opam-repository/pull/17616/files#diff-ec9c8838b3b6ff2cbb890f64386e483d40736ad8040844ce98c71a949930fc3fR10

Reply:
Thanks for the suggestion! Fixed in 19972619bff15e2eb97ae4767e5cb1fe6baa3ac6